docs/modules/ROOT/pages/how-tos/exoscale/install.adoc

```diff
@@ -391,34 +445,114 @@ terraform apply
 
 . Set up DNS NS records on parent zone using the data from the Terraform output variable `ns_records` from the previous step
 
-. Deploy bootstrap node
+. Create LB hieradata
 +
-[NOTE]
-====
-We initially provision VMs in "Stopped" state to ensure the private network interface is attached when the VM powers up.
-We provision the VMs in this two-step process to avoid VMs getting stuck in a state where they're unable to reach the machine config server when deploying the cluster into a private network.
```

Comment:
Is it not necessary to run terraform 3 times anymore?

Reply:
The whole "provision stopped, then boot" approach is only necessary when deploying the cluster into a private network.

Since the documentation currently doesn't discuss that option of the Terraform module, I've decided to drop the unnecessarily complicated terraform apply steps for now.